content/integrations/kubernetes.md

```diff
@@ -11,16 +11,276 @@ description: "{{< get-desc-from-git >}}"
 {{< img src="integrations/kubernetes/k8sdashboard.png" alt="Kubernetes Dashboard" responsive="true" >}}
```

Comment:
So we're not parsing readmes here?

Reply:
no, this article is to complex to be build on the fly from repos